### PR TITLE
[4338] initialized current address values, set input values to current values

### LINKED ISSUE #4338: Name of state saved for field without dropdown when user selected other country
**Description**:
Name of state saved for field without dropdown when user selected other country

**Expected behavior**:
Input field stay free when user selected other country

**Screenshots, Video, Logs**
https://vimeo.com/680872899/d8770b8c3f

**To Reproduce**:
1. Open FE
2. Login in account with saved address with state as dropdown
3. Click account > Address book > Edite address
4. Change country (Albania) on country Afganistan

**Actual result**:
Name of state saved for field without dropdown when user selected other country

### PR DESCRIPTION
**Pull request by pablobentivengo**
**Related issue(s):**
* Fixes #4338

**Problem:**
* Name of state saved for field without dropdown when user selected other country

**In this PR:**
* initialized current address values, set input values to current values

**Related PR(s):**
* #4359

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
3. Fixed but Can we stay empty State field when selecting 'Select country'? Now dropdown with States is displayed 
Screenshot 2022-03-07 at 09 16 16

File: packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js [hunk 1]
```diff
@@ -113,10 +113,30 @@ export class MyAccountAddressFormContainer extends PureComponent {
...
         const { countries, defaultCountry, address: { country_id: countryIdAddress } = {} } = this.props;
         const countryIdFixed = countryId || countryIdAddress || defaultCountry;
-
         return countries.find(({ value }) => value === countryIdFixed);
     }
 
+    getCurrentAddress() {
+        const { address } = this.props;
+
+        if (!address.id) {
```
Comment (riha112) [inline packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js:122]: better use:
```js
const { address, address: { id: addressId } = {} } = this.props;

if (!addressId) {
   ....
}
```

File: packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js [hunk 2]
```diff
@@ -113,10 +113,30 @@ export class MyAccountAddressFormContainer extends PureComponent {
...
+            return {
+                region: '',
+                regionId: 1,
+                zipCode: '',
+                city: ''
+            };
+        }
+        const { region: { region, region_id: regionId }, postcode, city } = address;
```
Comment (riha112) [inline packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js:130]: add empty line before `const { ...`

**Review by riha112 — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js (hunk 1 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js]: Done

File: packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js (hunk 2 above)
Comment (pablobentivengo) [reply packages/scandipwa/src/component/MyAccountAddressForm/MyAccountAddressForm.container.js:131]: Done

**Review by riha112 — APPROVED** [review]

**Pull request merged**